site/src/components/dismiss.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Dismiss
+description: The dismiss component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - dismiss
+---
+
+# Dismiss
+
+Dismiss is a subcomponent can be used within any other component that can be removed from the page. For example, modal, notification, banner, etc.
+
+## Usage
+
+Dismissal functionality requires two parts:
+
+1. The dismiss component, which has the `data-dismiss` attribute
+2. The parent component to dismiss, which has the `data-dismissable` attribute
+
+Below is an example usage with `notification` component.
+
+```html
+<div class="notification" id="example-notification-01" data-dismissable>
```

Comment:
Can we declutter this example but removing all but the most basic parts?

Reply:
Updated